### PR TITLE
Support for new API version

### PR DESCRIPTION
**Pull request by AvaisP**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,25 +64,30 @@ eventbuffer.setName('Test-Eeventbuffer-01');
 var options = {
   'timeIdentifier' : 'time',
   'timeFormat'     : 'iso_8601'
+  'thingIdentifier': 'motor'
```
Comment (phagunbaya) [inline README.md]: Options are not needed

File: lib/helper/models/Pipeline.js [hunk 2]
```diff
@@ -141,17 +134,6 @@ Pipeline.prototype.getThingName = function() {
   return _this.raw['singleThingID'];
 };
 
-Pipeline.prototype.setThingName = function(name) {
-  var _this = this;
-  _this.raw['singleThingID'] = name;
-  return _this;
-};
-
-Pipeline.prototype.getThingName = function() {
```
Comment (aniket-amrutkar) [inline lib/helper/models/Pipeline.js]: Don't remove getThingName

File: README.md [hunk 3]
```diff
@@ -60,29 +58,31 @@ var falkonry   = new Falkonry('https://service.falkonry.io', 'auth-token');
...
+        assessment.setName('Health')
+            .setInputSignals(['current', 'vibration', 'state']);
+
+        pipeline.setName('Pipeline-01')
+            .setEventbuffer(eventbuffer_id)
+            .setInputSignals(signals)
+            .setThingName('Motor')
+            .setAssessment(assessment);
+        return falkonry.createPipeline(pipeline, function(error, response){});
+    });
```
Comment (aniket-amrutkar) [inline README.md]: We should provide one test case for historian data

File: test/TestCreateEventbuffer.js [hunk 4]
```diff
@@ -35,78 +35,72 @@ describe.skip('Test Eventbuffer Creation', function(){
...
+    return falkonry.createEventbuffer(eventbuffer, function(error, response){
       assert.equal(error, null, 'Error creating Eventbuffer');
-
       if(!error) {
         eventbuffers.push(response);
         assert.equal(typeof response, 'object', 'Invalid Eventbuffer object after creation');
         assert.equal(typeof response.getId(), 'string', 'Invalid Eventbuffer object after creation');
         assert.equal(response.getName(), eventbuffer.getName(), 'Invalid Eventbuffer object after creation');
-        assert.equal(response.getSchemaList().length, 0, 'Invalid Eventbuffer object after creation');
+        assert.equal(response.getSchemaList().length, 1, 'Invalid Eventbuffer object after creation');
```
Comment (aniket-amrutkar) [inline test/TestCreateEventbuffer.js]: Check for time fields, data Path in webhook subscription

**Pull request merged**